dev-docs/FAQ.adoc

```diff
@@ -91,3 +91,9 @@ If you don't yet have an account, you have to ask for one in the 'users' or 'dev
 1. Please do not delete older files that you have already added - the complete history of an issue is important.
 1. The "Activity" section of an issue by default only lists "Comments". If you click on the "All" sub-tab you can see all activity related to this issue, including any edits that might have been made to the summary or description, as well as an commits that mention this issue in the commit log.
 1. Have follow up work to an existing issue that is closed?  If an issue is closed, its supposed to remain closed since it has been "shipped" in a release. Create a new issue and link it.
+
+=== Where can I find information about test history?
+
+* https://ge.apache.org/scans/tests?search.relativeStartTime=P90D&search.rootProjectNames=solr-root
```

Comment:
```suggestion
* https://ge.apache.org/scans/tests?search.relativeStartTime=P90D&search.rootProjectNames=solr*
```
Hope to drop the "-root" needless suffix, so using wildcards here to more future-proof this.

Reply:
Also I'm kind a shocked to see no reference added to Hossman's great work:
http://fucit.org/solr-jenkins-reports/failure-report.html
I'd actually list that *first* due to its longevity right now.

Reply:
combined suggestion

```suggestion
* http://fucit.org/solr-jenkins-reports/failure-report.html
* https://ge.apache.org/scans/tests?search.relativeStartTime=P90D&search.rootProjectNames=solr-*
```

Reply:
Why can't we simply edit the rootProject.name (in settings.gradle) to be "solr"?  Like apparently most other projects do, using the plain name of the project.  No need to to declare it to be a root or "grandparent".

Reply:
> Why can't we simply edit the rootProject.name (in settings.gradle) to be "solr"? Like apparently most other projects do, using the plain name of the project. No need to to declare it to be a root or "grandparent".

Interesting question. My focus here was/is to capture the https://lists.apache.org/thread/w15hfwdk22xdr535ndb9tbt42n4v9y52 mailing list info into the `dev-docs`.